validator/simple.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package validator
+
+import (
+	"context"
+	"fmt"
+	"reflect"
+
+	ga4gh "github.com/googlegenomics/ga4gh-identity"
+)
+
+// Simple is a ga4gh.Validator that compares the values in the incoming
+// identity to match the keys and values it contains.  For example, the Simple
+// validator: &Simple{"Role": "human"} would validate all identities containing
+// at least one Role with the value "human".  The Simple Validator does not
+// inspect the source of claims.
```

Comment:
Is it worth noting here that the values are not plain values but must be GA4GH Values?  Wasn't clear to me until I read the source for the Validate function.  Otherwise this will quietly fail if you provide a map of plain values.